code/__HELPERS/areas.dm

```diff
@@ -41,7 +41,7 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(/area/station/en
 			found_turfs += checkT // Since checkT is connected, add it to the list to be processed
 
 /**
- * Create an atmos zone (Think ZAS), similiar to [proc/detect_room] but it ignores walls and turfs which are non-[atmos_can_pass]
+ * Create an atmos zone (Think ZAS), similiar to PROC_REF(detect_room] but it ignores walls and turfs which are non-[atmos_can_pass])
```

Comment:
Your find and replace hit way too much documentation

Reply:
What i never understand is why people do this blindly so often.

every time i have regex replaced a thing, i manually hit replace next like 100 to 500 times before i hit replace all.

is that really so fucking hard.

Reply:
Yeah, i'll try to clean these up in a moment

Reply:
also we clearly need a spacemandmm system for regex code replacements. being able to pass in shit like --skip-comments or --only-comments or --proc-bodies-only would help so so so so much.

Reply:
yeah, would be nice. It's pain to do by hand.